src/client/pydaos/__init__.py

```diff
@@ -25,7 +22,7 @@ class PyDError(Exception):
     # possible.  There is an odd effect with daos_init() errors that
     # pydaos_shim is valid during __init__ but None during __str__ so format
     # the string early and just report it later on.
-    def __init__(self, message, rc):
+    def __init__(self, message, rc):  # pylint: disable=super-init-not-called
```

Comment:
@ashleypittman FYI - This one just came up locally, after not seeing it before. And this file hasn't been modified since I last merged master. So I wonder if more like this exist that just aren't being flagged